### ISSUE TITLE
about train loss

### ISSUE THREAD

**Issue by Sierkinhane** [tensorboy/pytorch_Realtime_Multi-Person_Pose_Estimation#78]
can you tell me the ultimate loss（train/val） in your training？：）

**Comment by FoolishMao** [comment]
what your train/val loss looks like?

**Comment by Sierkinhane** [comment]
@FoolishMao paf 0.0015， kpt 0.0010

**Comment by Sierkinhane** [comment]
@FoolishMao how about yours
